app/policies/user_policy.rb

```diff
@@ -0,0 +1,10 @@
+class UserPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+  def update?
+    user == record
```

Comment:
Does this need to be      user = record.user    ?

Reply:
Actually record is the current_user in Pundit terms. Doug told me